docs/hub/models-widgets-examples.md

```diff
@@ -1,5 +1,7 @@
 # Widget Examples
 
+Note that each widget example can also optionally describe the corresponding model output, direcly in the `output` property. See [the spec](./models-widgets#example-outputs) for more details.
+
```

Comment:
we can also update one of the examples on this page to have `output` as an example & add a hashlink to that example to make it clear.

For example, we can turn https://huggingface.co/docs/hub/models-widgets-examples#imagetoimage to have this optional `output` attribute

Reply:
yes, let's note to do that in a subsequent PR ok?